shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -1383,6 +1386,9 @@ - (void)dealloc {
   _activeView.textInputDelegate = nil;
   [_activeView release];
   [_inputHider release];
+  for (FlutterTextInputView* autofillView in _autofillContext.allValues) {
```

Comment:
This is just a precaution, in case the text input plugin is deallocated when there're still autofillable views alive.